src/ontology/cl-edit.owl

```diff
@@ -6823,7 +6823,7 @@ SubClassOf(obo:CL_0000378 obo:CL_0000630)
 # Class: obo:CL_0000379 (sensory processing neuron)
```

Comment:
This neuron lacks textual defs, references, etc..   I would also suggest adding references to support the logical definition when you add a textual definition - could also be a separate ticket

Reply:
I have seen you have already got a ticket for adding the defintion :)